demo/index.html

```diff
@@ -25,6 +26,11 @@
         <button class="js-dialog-btn-close">Close dialog 1</button>
     </div>
 
+    <div id="my-dialog2" class="dialog js-dialog">
```

Comment:
At the moment the role is added dynamically to ensure we add the correct one:
https://github.com/fedordead/vanilla-ui/blob/master/src/components/dialog/dialog.js#L259

We can move it out and leave it to the consumer to add but I thought it might be more robust the way I have it?